lib/fiddle/ffi_backend.rb

```diff
@@ -475,8 +475,11 @@ class Handle
     RTLD_NOW = FFI::DynamicLibrary::RTLD_NOW
 
     def initialize(libname = nil, flags = RTLD_LAZY | RTLD_GLOBAL)
-      @lib = FFI::DynamicLibrary.open(libname, flags) rescue LoadError
-      raise DLError.new("Could not open #{libname}") unless @lib
+      begin
+        @lib = FFI::DynamicLibrary.open(libname, flags)
+      rescue
```

Comment:
Plain `rescue` because it's LoadError on JRuby and RuntimeError for TruffleRuby.